plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -584,7 +626,7 @@ private static boolean isStruct(JsonNode typeNode)
 
     public static Map<String, Object> getColumnTypes(MetadataEntry metadataEntry)
     {
-        return getColumnProperties(metadataEntry, node -> OBJECT_MAPPER.convertValue(node.get("type"), new TypeReference<>(){}));
+        return getColumnProperties(metadataEntry, node -> OBJECT_MAPPER.convertValue(node.get("type"), new TypeReference<>() {}));
```

Comment:
This change is unrelated to "Introduce `data_skipping_stats_columns` table property in Delta Lake"